ExploreASL_Process.m

```diff
@@ -75,6 +75,15 @@
     %% 0 Run BIDS to Legacy
     x = xASL_init_Process(x);
     if ~x.opts.bSkipBIDS2Legacy
```

Comment:
I would use `bRun` rather than `bSkip` in general; but that's probably outside the scope of this hotfix